examples/echo-server.rs

```diff
@@ -41,6 +42,16 @@ async fn accept_connection(stream: TcpStream) {
 
     info!("New WebSocket connection: {}", addr);
 
-    let (write, read) = ws_stream.split();
-    read.forward(write).await.expect("Failed to forward message")
+    let (mut write, mut read) = ws_stream.split();
+    while let Some(msg) = read.next().await {
+        let resp = match msg.expect("Failed to read messages") {
+            msg @ (Message::Text(_) | Message::Binary(_)) => msg,
+            Message::Ping(data) => Message::Pong(data),
```

Comment:
we don't need to send a `Pong` as `tungstenite` [automatically does it for us](https://github.com/snapview/tungstenite-rs/blob/master/src/protocol/mod.rs#L172-L175)

Reply:
Oh, I see. Thanks for pointing it out.